apps/electric_cli/lib/electric_cli/session.ex

```diff
@@ -135,6 +135,9 @@ defmodule ElectricCli.Session do
     end
   end
 
+  def require_auth(true), do: :ok
```

Comment:
This reads counter-intuitive without the argument name: passing `true` to a function name `require_auth` actually skips auth. Without changing the implementation approach, the code can be made somewhat clearer by adding a body-less function head above this line:
```
def require_auth(local_stack?)
```

Reply:
Yup, thanks for the pointer :)